functions/manager.py

```diff
@@ -423,32 +423,65 @@ def _runtask(body: Dict[str, Union[str, None]]) -> Boto3Result:
     Keys:
         entrypoint: If set, the entryPoint command field in the ECS task
         definition will be changed to this value before the task is started.
+
+        container_id: The container to run as a new task.
+
+        service_id: The service where the container can be found.
+
+        cluster_id: The cluster where the service can be found.
+
+    Returns:
+        Boto3Result
     """
-    taskdef_entrypoint = body.get("entrypoint")
+    taskdef_entrypoint: str = body.get("entrypoint") or ""
+    if not isinstance(taskdef_entrypoint, str):
+        err_msg = {
+            "msg": "TypeError",
+            "data": "'entrypoint' key must be of type string",
+            "level": "critical",
+        }
+        log(**err_msg)
+        return Boto3Result(exc=TypeError(err_msg))
+
+    missing_required_keys: List[Optional[str]] = []
+    required_keys = {"container_id", "service_id", "cluster_id"}
```

Comment:
Is "container_id" really required? It looks like it's only used if an "entrypoint" argument is passed in.